### ISSUE TITLE
Rotten Tomato ratings not returned by default

### ISSUE THREAD

**Issue by chriscurtis** [plexinc-agents/PlexMovie.bundle#10]
In a default setup, users will use the "Freebase" agent and in the agent settings, TMDB will be higher priority than Freebase. With that setup, the user does _not_ receive Rotten Tomato ratings. (You have to move Freebase to a higher priority than TMDB in order to get them.) This doesn't look very good since we made a big deal about the RT support in the iOS app. :(

Worth noting that this is pretty a recent regression - had to be with either the 0.9.12.9 or .11 release.

**Comment by maxGreenblatt** [comment]
This is definitely not a recent regression as the current functionality is how this has always (been supposed to) worked. What you actually may have seen in the past was ratings that _looked_ like they came from RT but were actually from TMDb. It was a bug that we fixed a little while back (around 12.9 or 12.11).

That said - one-true-agent will fix this because users will have the ability to pick exactly what source their ratings, cast list, and plot summaries are taken from.

I'll leave this open until OTA is live.

**Comment by chriscurtis** [comment]
No, it's _definitely_ a recent regression in the end behavior, but maybe there's another cause for it, too. I've not changed my agent settings recently (well, prior to the testing I did for this issue earlier today). For me, movies added more than ~2 weeks ago have the RT ratings. If I refresh one of those now, the RT rating disappears. So, they _used_ to be picked up when I had the same agent settings and now they're not.

0.9.12.8 - works
0.9.12.11 - fails

**Comment by maxGreenblatt** [comment]
Hmm, ok - then can you get the XML of the same movie one for 12.8 and one for 12.11?

**Comment by maxGreenblatt** [comment]
Because, based on how the framework works - if TMDb is ahead of Freebase in your agent ordering and TMDb has a rating for a film, then the rating associated with the film should be TMDb's rating.

**Comment by maxGreenblatt** [comment]
plexinc/plex-ios-issues-new#174 is the bug that got fixed, that I believe is now causing the confusion where you think the old ratings were from RT, but they were actually from TMDb and just had RT's images

**Comment by chriscurtis** [comment]
No, I was looking in the XML and seeing the `audienceRating` parameter, which should only come from RT. And I totally tested and confirmed that .8 worked when I claimed that earlier. But, of course, now that I go to try again and get XML, I can't reproduce it. :/

But there obviously _has_ been a regression in something, somewhere. I mean, I've had TMDB as higher priority for ages and my content added to my library more than ~2 weeks ago has the RT `audienceRating` data. But newer stuff doesn't (or things refreshed more recently).

**Comment by maxGreenblatt** [comment]
Yes, that makes perfect sense, because there was a bug before plexinc/plex-ios-issues-new#174 - where TMDb didn't set `audienceRating` at all, so when the Framework tried to combine the metadata sources it would use TMDb for `rating` and RT for `audienceRating`. To stop that from happening we started having TMDb set `audienceRating` to null. Before we fixed the other issue people were complaining that the critics rating for RT was wrong which it was, because the critics rating was actually coming from TMDb. Does that make sense? 

Right now, If you want RT ratings you would need to have Freebase ahead of TMDb, unfortunately. But as I said - this will be fixed once we launch OTA.
